modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -205,6 +205,155 @@
+							},
+							element: 'div',
+						};
+
+						var style = new CKEDITOR.style(styleConfig);
+
+						editor.getSelection().lock();
+
+						Object.keys(this._styles).forEach((spacingKey) => {
+							var item = this._styles[spacingKey];
+
+							if (
+								this._checkActive({
+									elementPath: editor.elementPath(),
+									styleConfig: item.style,
+								})
+							) {
+								editor.removeStyle(
+									new CKEDITOR.style(item.style)
+								);
+							}
+						});
+
+						editor.applyStyle(style);
+
+						editor.getSelection().unlock();
+					},
+
+					_checkActive({elementPath, styleConfig}) {
+						var active = true;
+
+						if (elementPath && elementPath.lastElement) {
+							styleConfig.attributes.class
+								.split(' ')
+								.forEach((className) => {
+									active =
+										active &&
+										elementPath.lastElement.hasClass(
+											className
+										);
+								});
+						}
+						else {
+							active = false;
+						}
+
+						return active;
+					},
+
+					_styles: {
+						'1.0x': {
+							style: {attributes: {class: ''}, element: 'div'},
+						},
+						'1.5x': {
+							style: {
+								attributes: {class: 'mt-1 mb-1'},
+								element: 'div',
+							},
+						},
+						'2.0x': {
+							style: {
+								attributes: {class: 'mt-2 mb-2'},
+								element: 'div',
+							},
+						},
+						'3.0x': {
+							style: {
+								attributes: {class: 'mt-3 mb-3'},
+								element: 'div',
+							},
+						},
+						'4.0x': {
+							style: {
+								attributes: {class: 'mt-4 mb-4'},
+								element: 'div',
+							},
+						},
+						'5.0x': {
+							style: {
+								attributes: {class: 'mt-5 mb-5'},
+								element: 'div',
+							},
+						},
+					},
+
+					init() {
+						var instance = this;
+
+						Object.keys(this._styles).forEach((spacingKey) => {
+							instance.add(spacingKey, spacingKey);
+						});
+					},
+
+					label: '1.0x',
+
+					onClick(key) {
+						this._applyStyle(
+							this._styles[key].style.attributes.class
+						);
+
+						var richComboRef = editor.ui.get('LineSpacing');
+
+						richComboRef.select((item) => {
+							return item.text === key;
+						});
+					},
+
+					onRender() {
+						editor.on(
+							'selectionChange',
+							function (event) {
+								Object.keys(this._styles).forEach(
+									(spacingKey) => {
+										if (
+											this._checkActive({
+												elementPath: event.data.path,
+												styleConfig: this._styles[
+													spacingKey
+												].style,
+											})
+										) {
+											if (
+												this.getValue() !== spacingKey
+											) {
+												this.setValue(spacingKey);
+											}
+										}
+									}
+								);
+							},
+							this
+						);
+					},
+
+					panel: {
+						attributes: {'aria-label': 'LineSpacing'},
+						css: [CKEDITOR.skin.getPath('editor')].concat(
+							editor.config.contentsCss
+						),
+					},
+
+					title: 'Line Spacing',
```

Comment:
```suggestion
                                      title: 'Line Height',
```

Ideally we should use localized values, such as the one that are listed in `editor.lang`

Reply:
Hey @julien, I'm getting in trouble when doing it.  See https://github.com/liferay-frontend/liferay-portal/pull/1150#issuecomment-870976367